### ISSUE TITLE
[disscuss] Should we need something like QEMU_VERSION?

### ISSUE THREAD

**Issue by Weichen81** [kata-containers/runtime#615]
# Description of problem

In these days, we are fixing a bug https://github.com/kata-containers/runtime/issues/584 that is caused by QEMU max VCPU number. And I found the max VCPU number for different QEMU versions are different. For QEMU 2.11 or 2.12 stable version. On Arm64 host with GICv3 support, the QEMU can support to 123 VCPUs. But in latest upstream/master code, the QEMU support  123 + 512 VCPUs.
In this case, should we need an environment like QEMU_VERSION for us to make code be more flexible? Because we don't force user to use specific QEMU version.

@Pennyzct @jodh-intel @bergwolf

**Comment by devimc** [comment]
> Should we need something like QEMU_VERSION?

I don't think so, see: https://github.com/kata-containers/runtime/blob/master/virtcontainers/qemu.go#L82-L83
